website/src/pages/v1/serve/deployment/docker.mdx

```diff
@@ -2,11 +2,97 @@
 
 import { Callout } from '@theguild/components'
 
-<Callout>
-  This page is currently under construction and expected to change. Please feel free to reach out to
-  us directly in case you are having any troubles.
-</Callout>
-
 Docker is a tool that allows you to package an application and its dependencies into a container
 that can run on any system. This makes it easy to deploy applications in a consistent and
 reproducible way, regardless of the underlying infrastructure.
+
+## Running with Docker
```

Comment:
I think this documentation part should be split into 2 sections:

1. Running from a Docker image
2. Running from Docker image (with Hive)

Also, it should be clear that the user needs to create a config file, and mount it into the container.

Reply:
hmm...Laurin mentioned there's already a section for that in Hive's documentation that could integrate this part, but yes you're right mounting the config file isn't clear with the `docker run` usage and is only mentioned in the docker compose file.

Reply:
Yeah so let's add it to make sure it's clear how to use it with a volume mount for the config, for both Compose and `docker run`.

Reply:
I added the volume mount for a config, and it's available for both now